mwdb/web/src/components/Settings/SettingsView.js

```diff
@@ -125,7 +129,7 @@ export default function SettingsView(props) {
                     <div className="tab-content">
                         <Switch>
                             <AdministrativeRoute exact path="/admin">
```

Comment:
Regular <Route> should be used here with appropriate check on App.js level

Reply:
@KWMORALE: Fixed in https://github.com/CERT-Polska/mwdb-core/pull/366/commits/197575943664d8d405aeee901e25edc086a9c2e5